src/common/utils/resolve_reference.py

```diff
@@ -137,7 +127,9 @@ class AttributeItem:
     def __repr__(self):
         return f'Attribute="{self.path}"'
 
-    def resolve(self, document: dict | list, data_source: DataSource, get_data_source: Callable) -> Tuple[dict, str]:
+    def resolve(
+        self, document: dict | list | None, data_source: DataSource, get_data_source: Callable
+    ) -> tuple[dict | list | None, str]:
         try:
             result = find(document, [self.path])
```

Comment:
find does not accept document of type None

Reply:
fixed in https://github.com/equinor/data-modelling-storage-service/pull/435